sunpy/image/tests/test_transform.py

```diff
@@ -245,6 +245,7 @@ def test_float32(identity):
     assert np.issubdtype(out_arr.dtype, np.float32)
 
 
+@skip_windows
```

Comment:
This is now broken on windows and I don't have the skills to debug why.

Reply:
I should open an issue to track this.

Reply:
This makes me nervous but ok

Reply:
We will have to track this down but its a numpy issue on windows. Nothing we can do to fix that outside of reporting it upstream.